.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -24,6 +24,17 @@ body:
         4. See error
     validations:
       required: true
+ - type: markdown
+    attributes:
+      value: |
+        # An image might be worth 100 words! a video would be worth tons!
+        This will help us to get a better understanding of the issue.
+  - type: textarea
+    attributes:
+      label: Do you have any images and screen captures? 
+      description: A clear image showing the bad behavior would be awesome, if not available write N/A
+    validations:
+      required: true
```

Comment:
Not sure if this should be required since it's a heavy lift to do this kind of image capture. I'd recommend this be `false`.

Reply:
I kept it required since it would be better to have images of the error or the issue that the person is facing otherwise it might be hard to understand what they went through.